developer/src/tike/xml/help/contexthelp.xml

```diff
@@ -182,7 +182,20 @@
     <Control Name="cmdTestKeymanWeb" Title="Test KeymanWeb Keyboard">
       <p>Tests your KeymanWeb keyboard in an Internet Explorer embedded window</p>
     </Control>
+    <Control Name="pagesTouchLayout" Title="Touch Layout page">
+      <p>Touch Layout tab defines the functionality of the keyboard for mobile devices:
+      <ul>
+        <li>Top bar Controls</li>
+        <li>Keyboard area</li>
+        <li>Middle bar controls</li>
+        <li>Long press area</li>
+      </ul>
```

Comment:
This is probably not helpful documentation. Is it copied from help.keyman.com?

Reply:
I couldn't find it in KD 17.0 or help.keyman.com, so I followed a similar pattern of "pages" in ControlName and added the content as seen. It is possible that this section is from an older version, but I haven't found exactly where it is either.

Reply:
OK, I am not sure where to go with this. It doesn't seem worth writing more detailed documentation here, so perhaps we need to update the help site and then copy that over?

Reply:
I'm happy to work on that. I hope I can find it in version 15.0 or so, and then update the documents there. If users are still using older versions, they will see the context help.